src/main/java/com/netchill/webservices/api/GenreWs.java

```diff
@@ -3,33 +3,34 @@
 import com.coreoz.plume.jersey.security.permission.PublicApi;
 import com.netchill.db.generated.Genre;
 import com.netchill.services.configuration.ConfigurationService;
-import com.netchill.services.general.GeneralService;
+import com.netchill.services.genre.GenreService;
 import io.swagger.v3.oas.annotations.tags.Tag;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
 import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
 
-@Path("")
-@Tag(name = "General", description = "things that are same in movie and serie")
+@Path("/")
```

Comment:
le path n'est pas bon 
```suggestion
@Path("/Genre")
```

Corriger également les paths enfants.

Reply:
Super, tu l'as déjà fait dans la prochaine MR !